optimizer/src/main/java/com/netease/arctic/optimizer/flink/FlinkConsumer.java

```diff
@@ -67,7 +72,7 @@ public void run(SourceContext<TaskWrapper> sourceContext) throws Exception {
           retry = 0;
           LOG.error("flink source has tried too many times, and the subscription message is suspended." +
               " Please check for errors");
-          Thread.sleep(10000);
+          Thread.sleep(2 * POLL_INTERVAL);
```

Comment:
Is it possible to wait uniformly if no task is obtained or if the task fails to be obtained?

Reply:
Yes, I will modify it.